src/style.css

```diff
@@ -1 +1,77 @@
 /* Styles go here */
+* {
```

Comment:
Check the [checklist #9](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md), using * is not recommended